src/main/resources/fabric.mod.json

```diff
@@ -12,7 +12,7 @@
     "discord": "https://discord.gg/viaversion"
   },
   "depends": {
-    "fabricloader": ">=0.15.11",
+    "fabricloader": ">=0.16.2",
```

Comment:
Is there any reason why we should enforce newer loader versions?

Reply:
Basically fixes version detection here, though don't merge this yet until 1.21.2 is finalized.

Reply:
I don't see how this could help end users? If there aren't any relevant changes for users, there is no need to bump the required dependency version. The build script change can stay.

Reply:
Well we kinda have to as the Fabric API for upcoming 1.21.2 requires a much higher fabric loader version now, though i still wonder if it's even possible to make viafabric without it / bare minimum.

![a](https://github.com/user-attachments/assets/b68d84ba-9c97-499e-98ea-5c49827b7f3b)

Reply:
I wonder why we even have fabricloader listed in our requirements if fabric api is already enforcing a correct version?

Reply:
This is actually quite common in most fabric mods out there where they would tell the end-user that older fabric loaders are not supported as an example, but i can remove this in viafabric if you think this is unwanted.

Reply:
I would just remove the fabric loader from our dependency list since we don't rely on any version-specific features. 
Additionally, each fabric api submodule has the correct loader dependency , see [here](https://github.com/FabricMC/fabric/blob/1.21.2/fabric-resource-loader-v0/src/main/resources/fabric.mod.json)

Reply:
Removed.